tests/test_compound.py

```diff
@@ -0,0 +1,116 @@
+from typing import List
+
+from mev_inspect.compound_liquidations import get_compound_liquidations, get_all_comp_markets
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.schemas.classified_traces import Protocol
+from mev_inspect.classifiers.trace import TraceClassifier
+from tests.utils import load_test_block
+from web3 import Web3
+import os
+
+rpc = os.getenv("RPC_URL")
+
+if rpc is None:
+    raise RuntimeError("Missing environment variable RPC_URL")
+else:
+    w3 = Web3(Web3.HTTPProvider(rpc))
+
+
+def test_c_ether_liquidations():
+    block_number = 13234998
+    transaction_hash = "0x78f7e67391c2bacde45e5057241f8b9e21a59330bce4332eecfff8fac279d090"
+
+    liquidations = [
+        Liquidation(
+            liquidated_user="0xb5535a3681cf8d5431b8acfd779e2f79677ecce9",
+            liquidator_user="0xe0090ec6895c087a393f0e45f1f85098a6c33bef",
+            collateral_token_address="0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+            debt_token_address="0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+            debt_purchase_amount=268066492249420078,
+            received_amount=105283041640,
+            protocol=Protocol.compound_v2,
+            transaction_hash=transaction_hash,
+            trace_address=[1],
+            block_number=block_number,
+        )
+    ]
+    block = load_test_block(block_number)
+    trace_classifier = TraceClassifier()
+    classified_traces = trace_classifier.classify(block.traces)
+    comp_markets = get_all_comp_markets(w3)
```

Comment:
similar to saving the block data, is it possible to save this data to make tests fully static?

otherwise RPC / HTTP issues could cause these tests to fail when the code itself is working fine

also prevents us from getting that sweet sweet robert airplane code ✈️